custom/ewsghana/handlers/alerts.py

```diff
@@ -293,8 +293,11 @@ def handle(self):
             text = self.msg.text
 
         if not domain.commtrack_enabled:
-            print "tutaj?"
             return False
+
+        if not verified_contact.location_id:
+            self.respond(NO_SUPPLY_POINT_MESSAGE)
+
```

Comment:
In this handler it should stop here because we can't save report without location.